backend/app/models/business.py

```diff
@@ -0,0 +1,34 @@
+"""
+    app.models.business
+    ~~~~~~~~~~~~~~~~~~~
+    Defines the Pydantic model for the Business resource
+"""
+
+from pydantic import BaseModel
+from typing import Optional
+
+
+class Business(BaseModel):
```

Comment:
not sure on the naming here, while business is a representation of the "business"s meta, i believe we should name it something else. e.g. Enterprise, Company, Entity, etc....

Reply:
Ill rename to company